pkg/apic/apiserver/models/management/v1alpha1/model_traceability_agent_spec.go

```diff
@@ -9,10 +9,10 @@
 
 package v1alpha1
 
-// TraceabilityAgentSpec struct for TraceabilityAgentSpec
+// TraceabilityAgentSpec  (management.v1alpha1.TraceabilityAgent)
 type TraceabilityAgentSpec struct {
 	// The dataplane type that this agent connects to
-	DataplaneType string                      `json:"dataplaneType"`
-	Config        TraceabilityAgentSpecConfig `json:"config"`
-	Logging       DiscoveryAgentSpecLogging   `json:"logging,omitempty"`
+	DataplaneType string                       `json:"dataplaneType"`
+	Config        TraceabilityAgentSpecConfig  `json:"config"`
+	Logging       TraceabilityAgentSpecLogging `json:"logging,omitempty"`
```

Comment:
Can we check that any reference to the TraceabilityAgent resource is updated for this change?

Reply:
it looks like it is...